src/server/system_services/stats_aggregator.js

```diff
@@ -569,7 +569,9 @@ function get_nodes_stats(req) {
                     account_id: support_account._id
                 })
             }))
-        .spread((nodes_results, hosts_results) => {
+        .then(res => {
+            let nodes_results = res[0];
+            let hosts_results = res[1];
```

Comment:
```suggestion
        .then(([nodes_results, hosts_results])=> {
```

Reply:
when doing that we get a lint error:
`Unexpected block statement surrounding arrow body; move the returned value immediately after the =>.eslint(arrow-body-style)`